### PR TITLE
Update forseti server to pull most configuration from the server config.

### PR DESCRIPTION
**Pull request by ahoying**
* Remove flags --gsuite_private_keyfile, --gsuite_admin_email, --root_resource_id
* Add inventory: section to server config
  * Add root_resource_id to server config under inventory
  * Group api quota configurations under new api_quota key under inventory.
* Parse server config on server startup and store sections in service_config.
* Remove unused main() from scanner and notifier.

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] My PR has been functionally tested.
- [x] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass.
- [x] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ahoying** [comment]
Closes #1135 and #1035.

File: setup/gcp/installer/forseti_server_installer.py [hunk 1]
```diff
@@ -188,7 +186,8 @@ def get_configuration_values(self):
             'EMAIL_SENDER': self.config.notification_sender_email,
             'SENDGRID_API_KEY': self.config.sendgrid_api_key,
             'SCANNER_BUCKET': bucket_name[len('gs://'):],
-            'DOMAIN_SUPER_ADMIN_EMAIL': self.config.gsuite_superadmin_email
+            'DOMAIN_SUPER_ADMIN_EMAIL': self.config.gsuite_superadmin_email,
```
Comment (joecheuk) [inline setup/gcp/installer/forseti_server_installer.py]: I think the key here should be 'GSUITE_ADMIN_EMAIL' instead, that's what we are using in our deployment template.

File: configs/server/forseti_conf_server.yaml.in [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+    # Root resource to start crawling from, formatted as
+    # <resource_type>/<resource_id>, (e.g. "organizations/12345677890")
+    root_resource_id: {ROOT_RESOURCE_ID}
+
+    # gsuite access
+    groups_service_account_key_file: /home/ubuntu/gsuite_key.json
+    domain_super_admin_email: {DOMAIN_SUPER_ADMIN_EMAIL}
+
+    api_quota:
     # The pre-populated values are the defaults from GCP.
```
Comment (blueandgold) [inline configs/server/forseti_conf_server.yaml.in]: really small nit, not a blocker: seems like this line should be shift too

File: setup/gcp/installer/forseti_server_installer.py (hunk 1 above)
Comment (ahoying) [reply setup/gcp/installer/forseti_server_installer.py]: Here is the line in the deployment template:

domain_super_admin_email: {DOMAIN_SUPER_ADMIN_EMAIL}

File: configs/server/forseti_conf_server.yaml.sample [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+##############################################################################
+
+inventory:
+
+    # Root resource to start crawling from, formatted as
+    # <resource_type>/<resource_id>, (e.g. "organizations/12345677890")
+    root_resource_id: ROOT_RESOURCE_ID
+
+    # gsuite
+    groups_service_account_key_file: GROUPS_SERVICE_ACCOUNT_KEY_FILE
```
Comment (blueandgold) [inline configs/server/forseti_conf_server.yaml.sample]: nit: can we change both of these to `gsuite_service_account_key_file`; group wouldn't be accurate as we are also using the same key to get users.

File: setup/gcp/installer/forseti_server_installer.py (hunk 1 above)
Comment (ahoying) [reply setup/gcp/installer/forseti_server_installer.py]: Fixed by removing GSUITE_ADMIN_EMAIL from deployment template.

File: configs/server/forseti_conf_server.yaml.in (hunk 2 above)
Comment (ahoying) [reply configs/server/forseti_conf_server.yaml.in]: Done.

File: configs/server/forseti_conf_server.yaml.sample (hunk 3 above)
Comment (ahoying) [reply configs/server/forseti_conf_server.yaml.sample]: Done.

File: setup/gcp/scripts/initialize_forseti_services.sh [hunk 4]
```diff
@@ -30,25 +30,22 @@ set -o nounset
 # script will fail immediately rather than partially succeeding.
 echo "Cloud SQL Instance Connection string: ${SQL_INSTANCE_CONN_STRING}"
 echo "SQL port: ${SQL_PORT}"
-echo "Forseti DB name: ${FORSETI_DB_NAME}"
-echo "G Suite admin email: ${GSUITE_ADMIN_EMAIL}"
-echo "Root resource ID: ${ROOT_RESOURCE_ID}"
 if ! [[ -f $GSUITE_ADMIN_CREDENTIAL_PATH ]]; then
```
Comment (joecheuk) [inline setup/gcp/scripts/initialize_forseti_services.sh]: I think we no longer have variable $GSUITE_ADMIN_CREDENTIAL_PATH set during the VM initialization.

Message I got during the init: 
./setup/gcp/scripts/initialize_forseti_services.sh: line 33: GSUITE_ADMIN_CREDENTIAL_PATH: unbound variable

Comment (ahoying) [reply setup/gcp/scripts/initialize_forseti_services.sh]: Thanks, fixed.

**Pull request merged**